### PR TITLE
Add guide for storing sensitive information

### PR DESCRIPTION
**Pull request by patcon**
This is how we've dealt with first SINs, and then completed TD1 and TD1ON tax forms.

Does this feel like the right format/place to mention this?

### CODE REVIEW

File: guides.md [hunk 1]
```diff
@@ -19,6 +19,7 @@ our values.
 
 - [Meetings](#meetings)
 - [Shortlinks](#shortlinks)
+- [Storage: Sensitive Data](#storage-sensitive-data)
```
Comment (benhylau) [inline guides.md]: Maybe the `Storage: ` makes the index looks confusing?

```suggestion
- [Sensitive Data](#storage-sensitive-data)
```

File: guides.md [hunk 2]
```diff
@@ -127,6 +128,16 @@ Instructions:
 [Chrome](https://www.techrepublic.com/article/pro-tip-add-custom-search-engines-in-chrome-for-more-efficient-searching/)
 | [Firefox (requires extension)](https://addons.mozilla.org/en-US/firefox/addon/add-custom-search-engine/)
 
+## Storage: Sensitive Data
```
Comment (benhylau) [inline guides.md]:
```suggestion
## Sensitive Data
```

File: guides.md [hunk 3]
```diff
@@ -127,6 +128,16 @@ Instructions:
...
+
+In general, most staff prefer not to use third-party SaaS services, like Google Drive, for storing sensitive information.
+
+We store **short strings** (like SINs or codes) as individual credentials in Passbolt. These can be shared with both the providing individual and a privileged group that requires access, like Finance WG.
+
+If we must **store sensitive docs** (like PDFs) on GDrive for convenience, we first encrypt them with a password. We keep this in Passbolt as entry "GDrive: File Encryption". All employees can access this password. Simpler encryption schemes are preferred, for example, default PDF encryption.
```
Comment (benhylau) [inline guides.md]: Should we use generic `Drive` instead of `GDrive` in case we change the Drive solution in the future?

File: guides.md (hunk 1 above)
Comment (patcon) [reply guides.md]:
```suggestion
- [Sensitive Data](#sensitive-data)
```

File: guides.md (hunk 3 above)
Comment (patcon) [reply guides.md]:
```suggestion
If we must **store sensitive docs** (like PDFs) in a shared drive for convenience (e.g., Google Drive),
we first encrypt them with a password.
We keep this in Passbolt under the entry "Shared Drive: File Encryption".
All employees can access this password.
Simpler encryption schemes are preferred, for example, default PDF encryption.
```

this work?

Comment (patcon) [reply guides.md]: Maybe we should recommend `some-file.encrypted.pdf` or `some-file.enc.pdf`

File: guides.md [hunk 4]
```diff
@@ -269,6 +270,21 @@ Instructions:
 [Chrome](https://www.techrepublic.com/article/pro-tip-add-custom-search-engines-in-chrome-for-more-efficient-searching/)
 | [Firefox (requires extension)](https://addons.mozilla.org/en-US/firefox/addon/add-custom-search-engine/)
 
+## Sensitive Data
+
+In general, most staff prefer not to use third-party SaaS services, like Google Drive, for storing sensitive information.
+
+We store **short strings** (like SINs or codes) as individual credentials in Passbolt. These can be shared with both the providing individual and a privileged group that requires access, like Finance WG.
```
Comment (patcon) [inline guides.md]: Re-read made it seem confusing. Leaning on passbolt terminology.

```suggestion
We store **short strings** (like SINs or codes) as password entries in Passbolt. These entries should be shared with both a privileged _group_ that needs access (e.g., Finance WG) and the _individual_.
```

**Comment by patcon** [comment]
Thanks both!

**Pull request merged**